tests/integration-tests/requirements.txt

```diff
@@ -1,13 +1,17 @@
+../../api/client/src
```

Comment:
Where are we using api client src code?

Reply:
I did not add this dependency and my changes do not require it.
I just moved it according to isort policy